src/core/index.ts

```diff
@@ -337,11 +337,14 @@ export class ThirdwebSDK {
 
   /**
    *
-   * @param address - The contract address of the given Collection module.
-   * @returns The Collection Module.
+   * @param address - The contract address of the given Bundle module.
+   * @returns The Bundle Module.
    */
-  public getCollectionModule(address: string): CollectionModule {
-    return this.getOrCreateModule(address, CollectionModule);
+  public getCollectionModule(address: string): BundleModule {
```

Comment:
I think we should type alias this. Otherwise, it'd breaking changes for typescripts.